### PR TITLE
marcioanatrielo/c02-network08

### PR DESCRIPTION
**Pull request by marcioanatrielo**
# Description

Exercise (C02-NETWORK08): 

Student Name: Marcio Andre

> A brief explanation of what has been done.

Step 1: Got to Network ACL

Step 2: Select Network ACL ID - acl-0307abe60fb4d940a

Step 3: Go to Action  

Step 4: Select edit inbound rule 

Step 5: Add rule 100 Source: VPC Network - Allow   

Step 6: Add rule 101 Source: 0.0.0.0/0 - Allow 

Step 7: Add rule 102 Source: 0.0.0.0/0 - Deny 

Step 8: Save 

Step 9: Go to Action  

Step 10: Select edit outbound rule 

Step 11: Add rule 100 Source: VPC Network - Allow   

Step 12: Add rule 101 Source: 0.0.0.0/0 - Allow 

Step 13: Add rule 102 Source: 0.0.0.0/0 - Deny 

Step 14: Save 

Step 15: Create inbound Rule 10 to block SSH from 172.16.200.0/24

Step 16: Save 

Step 17: Create outbound Rule 10 to block access from 180.200.50.0/24

Step 18: Save 

## Type of change

- [X] Exercise Submission (one exercise per PR)
  - [X] This PR is just for **one** exercise of a class
  - [X] PR name follows the pattern `<github-username>/<exercise-number>`
  - [X] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [X] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/02class/exercises/c02-network08/marcio-anatrielo/c02-network08/c02-network08.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+# Answer for exercise c02-network08 
+
+What is the difference between inbound x outbound? Becouse this FW is stateless will be necessary allow or deny bouth direcition for all connection wich go to our network.
+What is going to happen by default? It is allowed.   Everything allowed or denied? Base on rule number for the last rule on this exemple, the traffic can be allow or deny in my case it is allow becouse the rule number is 102 with will no be consider when new connection hit this firewall. 
+How would you block SSH connection comming from the network 172.16.200.0/24? 
+
+Create a new Inbound Rule:  
+Rule: 10
```
Comment (caiocezart) [inline classes/02class/exercises/c02-network08/marcio-anatrielo/c02-network08/c02-network08.txt]: are you sure that the number can be lower than 100 on this example?

File: classes/02class/exercises/c02-network08/marcio-anatrielo/c02-network08/c02-network08.txt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Type: SSH (22)
+Protocol: TCP (6)
+Port: 22
+Source: 172.16.200.0/24
+Action: DENY
+
+How would you block access from your network to an external network 180.200.50.0/24?
+
+Create a new Outbound Rule:
+Rule: 10
```
Comment (caiocezart) [inline classes/02class/exercises/c02-network08/marcio-anatrielo/c02-network08/c02-network08.txt]: are you sure that the number can be lower than 100 on this example?

File: classes/02class/exercises/c02-network08/marcio-anatrielo/c02-network08/c02-network08.txt [hunk 3]
```diff
@@ -0,0 +1,25 @@
+
+# Answer for exercise c02-network08 
+
+What is the difference between inbound x outbound? Becouse this FW is stateless will be necessary allow or deny bouth direcition for all connection wich go to our network.
+What is going to happen by default? It is allowed.   Everything allowed or denied? Base on rule number for the last rule on this exemple, the traffic can be allow or deny in my case it is allow becouse the rule number is 102 with will no be consider when new connection hit this firewall.
```
Comment (caiocezart) [inline classes/02class/exercises/c02-network08/marcio-anatrielo/c02-network08/c02-network08.txt]: I need you to rewrite this. I can't really understand

File: classes/02class/exercises/c02-network08/marcio-anatrielo/c02-network08/c02-network08.txt [hunk 4]
```diff
@@ -0,0 +1,25 @@
+
```
Comment (kikobr82) [inline classes/02class/exercises/c02-network08/marcio-anatrielo/c02-network08/c02-network08.txt]: file is in the wrong folder. it should be your github username.

File: classes/02class/exercises/c02-network08/marcio-anatrielo/c02-network08/c02-network08.txt (hunk 2 above)
Comment (marcioanatrielo) [reply classes/02class/exercises/c02-network08/marcio-anatrielo/c02-network08/c02-network08.txt]: If I don´t check this role first them 100 that police will not be consider by my firewall
.

File: classes/02class/exercises/c02-network08/marcio-anatrielo/c02-network08/c02-network08.txt (hunk 1 above)
Comment (marcioanatrielo) [reply classes/02class/exercises/c02-network08/marcio-anatrielo/c02-network08/c02-network08.txt]: If it is not lower them 100 that role will no be match by the firewall.

**Pull request closed without merging**